launcher/ui/setupwizard/ThemeWizardPage.ui

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ThemeWizardPage</class>
+ <widget class="QWizardPage" name="ThemeWizardPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>510</width>
+    <height>552</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>WizardPage</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout_2">
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Select the Theme you wish to use</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="ThemeCustomizationWidget" name="themeCustomizationWidget" native="true">
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>100</height>
+      </size>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="label_2">
+     <property name="text">
+      <string>Hint: The cat appears in the background and does not serve a purpose, it is purely visual.</string>
+     </property>
```

Comment:
```suggestion
     <property name="text">
      <string>Hint: The cat appears in the background and is not shown by default. It is only made visible when pressing the Cat button in the Toolbar.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
```

I think this makes things a bit clearer, but maybe it could be improved too.